### DISCUSSION TITLE
Vim / Neovim Dictionary

### DISCUSSION THREAD

**Discussion by kevintraver** [streetsidesoftware/cspell-dicts#2613] [Ideas]
I think it might make sense to create a new dictionary for working with vim/neovim development and plugins for words that don't really belong anywhere else.

Here is a brief starter list:

- augroup
- autocmd
- bufname
- bufnr
- cmdline
- deepcopy
- diffview
- feedkeys
- fnamemodify
- keymap
- keymaps
- lnum
- lspconfig
- neovim
- nvim
- quickfix
- statusline
- tabpage
- winid

**Comment by Jason3S** [comment]
@kevintraver,

That is fine.

**Reply by kevintraver** [reply]
Ok great! I will set up a new dictionary

**Reply by kevintraver** [reply]
I will take some time gathering a larger list to avoid creating excessive PRs 😉

**Reply by kevintraver** [reply]
Ok, I have a pretty good initial list now. What is the process for starting a new dictionary, or is there documentation somewhere?

**Reply by Ki-er** [reply]
@kevintraver You can run the create dictionary script.

**Reply by kevintraver** [reply]
https://github.com/streetsidesoftware/cspell-dicts/pull/2664
